Libraries/WebSocket/WebSocket.js

```diff
@@ -245,7 +245,12 @@ class WebSocket extends (EventTarget(...WEBSOCKET_EVENTS): any) {
             data = BlobManager.createFromOptions(ev.data);
             break;
         }
-        this.dispatchEvent(new WebSocketEvent('message', {data}));
+        const raw_length = ev.raw_length;
+        if (raw_length == null) {
+          this.dispatchEvent(new WebSocketEvent('message', {data}));
+        } else {
+          this.dispatchEvent(new WebSocketEvent('message', {data, raw_length}));
```

Comment:
consider unconditionally including the field; it being `undefined` should be sufficient (accessing a non-existing field results in the same value); conditional presence seems complicated for little benefit

Reply:
Done, although GatewaySocket still has to treat it as conditionally present because it handles both RN and web/desktop